app/views/idv/doc_auth/_combined_upload.html.erb

```diff
@@ -0,0 +1,76 @@
+<% title t('titles.doc_auth.upload') %>
+
+<%= render 'idv/doc_auth/error_messages', flow_session: flow_session %>
+
+<%= render PageHeadingComponent.new do %>
+    <%= t('doc_auth.headings.combined_upload') %>
+<% end %>
+
+<p>
+    <%= t('doc_auth.info.combined_upload') %>
+</p>
+
+<hr class="margin-y-4" />
+
```

Comment:
I don't see this horizontal rule in the design? Should it be here? (Am I looking at the wrong design?)

```suggestion
```